integration-cli/docker_cli_events_test.go

```diff
@@ -250,6 +250,44 @@ func (s *DockerSuite) TestEventsImageImport(c *check.C) {
 	c.Assert(matches["action"], checker.Equals, "import", check.Commentf("matches: %v\nout:\n%s\n", matches, out))
 }
 
+func (s *DockerSuite) TestEventsImageLoad(c *check.C) {
+	testRequires(c, DaemonIsLinux)
+	dockerCmd(c, "pull", "busybox")
+	since := daemonUnixTime(c)
+
+	out, _ := dockerCmd(c, "images", "-q", "--no-trunc", "busybox")
+	longImageID := strings.TrimSpace(out)
+	c.Assert(longImageID, checker.Not(check.Equals), "", check.Commentf("Id should not be empty"))
+
+	dockerCmd(c, "save", "-o", "saveimg.tar", "busybox")
+	dockerCmd(c, "rmi", "busybox")
```

Comment:
This code removes busybox. Meaning it will be repulled from instead of using the frozen image. Image pulled from hub doesn't have a parent chain.